PeerioTechnologies/peerio-desktop
# chore: 'add member' userpicker search context

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/8653/can-t-add-patients-to-patient-rooms-after-creation
https://github.com/PeerioTechnologies/peerio-icebear/pull/217

#### Testing instructions  
userpicker in "add member" (from any room) should use correct search context now. (it was working before everywhere except Patient Rooms, now it should work there too)

will merge this once icebear branch is merged

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
